pyirf/io/gadf.py

```diff
@@ -268,7 +268,7 @@ def create_rad_max_hdu(
     """
     Create a fits binary table HDU in GADF format for the directional cut.
     See the specification at
-    https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/aeff/index.html
+    https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/index.html?highlight=RAD_MAX#rad-max
```

Comment:
apparently you copy-paste the url while you were searching for "RAD_MAX"

it should be just

https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/index.html#rad-max